tests/Translators/RoadConditions/test_tim_generator.py

```diff
@@ -0,0 +1,61 @@
+from Translators.RoadConditions import tim_generator
+
+############################ getItisCodes ############################
+def test_get_itis_codes_no_conditions():
+    feature = {
+        'properties': {
+            'currentConditions': []
+        }
+    }
+    itisCodes = tim_generator.get_itis_codes(feature)
+    assert itisCodes == []
+
+def test_get_itis_codes_single_condition():
+    feature = {
+        'properties': {
+            'currentConditions': [
+                {'conditionDescription': 'accident on right portion of road'}
+            ]
+        }
+    }
+    itisCodes = tim_generator.get_itis_codes(feature)
+    assert itisCodes == ['513']  # ItisCodes.ACCIDENT.value
```

Comment:
Would it make sense to import the ItisCodes and reference them that way instead of by text string?

Reply:
Updated to use the enum instead of string values

Reply:
Checked out another branch to force it to refresh things and it's back to referencing the ITIS codes properly 🤷‍♂️